### PR TITLE
Add Boxel icon to left of Submode Switcher menu

### PR DESCRIPTION
**Pull request by FadhlanR**
https://github.com/user-attachments/assets/867a29db-7443-4100-b694-93d2a60f5989

### CODE REVIEW

**Review by lukemelia — APPROVED** [review]

File: packages/host/app/components/operator-mode/submode-layout.gts [hunk 1]
```diff
@@ -315,12 +350,22 @@ export default class SubmodeLayout extends Component<Signature> {
...
+
+        display: flex;
+        align-items: center;
+        gap: var(--boxel-sp-xs);
+      }
+
+      .boxel-title {
+        color: var(--boxel-light);
+        font: 600 var(--boxel-font-lg);
+        letter-spacing: var(--boxel-lsp-xxl);
```
Comment (backspace) [inline packages/host/app/components/operator-mode/submode-layout.gts]: Maybe this needs a tiny bit of space on the left and top to match the design?

V1 Dashboard – 1 - Generative Software (Work in Progress) - Zeplin 2024-08-15 12-15-14

vs the implementation where it’s a bit too close and slightly off-centre vertically

Boxel 2024-08-15 12-15-30

File: packages/boxel-ui/addon/raw-icons/dark-boxel-icon.svg
Comment (backspace) [inline packages/boxel-ui/addon/raw-icons/dark-boxel-icon.svg:1]: I don’t know if it’s important but could the existing icon be parameterised with colour variables to allow it to be used for this as well?

**Review by backspace — APPROVED** [review]
This looks good, and it’ll be useful for me in #1490 as a place to put the unformatted-listing of realms 🎉

I know it’s not part of the design but I do have a UX concern that there could be a close button in case people are confused about how to get out of the overlay. But maybe it’s obvious that the button is a toggle, since that’s how they got into it.

A very-brief fade-in animation when the overlay opens might add some smoothness 🤔

File: packages/host/app/components/operator-mode/submode-layout.gts (hunk 1 above)
Comment (burieberry) [reply packages/host/app/components/operator-mode/submode-layout.gts]: It looks like the font for Boxel is different in the design. Rustica Black font-weight 900 letter-spacing 3px. The gap between logo and Boxel is 15px (--boxel-sp-sm)

File: packages/boxel-ui/addon/raw-icons/dark-boxel-icon.svg
Comment (FadhlanR) [reply packages/boxel-ui/addon/raw-icons/dark-boxel-icon.svg:1]: The structure is a bit different: in the `dark-boxel-icon`, the lines are colored cyan, while in the `boxel-icon`, they are transparent.

**Comment by FadhlanR** [comment]
> This looks good, and it’ll be useful for me in #1490 as a place to put the unformatted-listing of realms 🎉
> 
> I know it’s not part of the design but I do have a UX concern that there could be a close button in case people are confused about how to get out of the overlay. But maybe it’s obvious that the button is a toggle, since that’s how they got into it.
> 
> A very-brief fade-in animation when the overlay opens might add some smoothness 🤔

I added the animation, now it looks like this when opening workspace chooser.


https://github.com/user-attachments/assets/4d6d94ce-98ae-46fb-8098-697158ff3483

File: packages/host/app/components/operator-mode/submode-layout.gts (hunk 1 above)
Comment (FadhlanR) [reply packages/host/app/components/operator-mode/submode-layout.gts]: Updated

File: packages/boxel-ui/addon/raw-icons/dark-boxel-icon.svg
Comment (FadhlanR) [reply packages/boxel-ui/addon/raw-icons/dark-boxel-icon.svg:1]: Updated

**Review by burieberry — APPROVED** [review]
Nice animation

**Pull request merged**